src/stylesheets/common/_styleguide.scss

```diff
@@ -1,3 +1,11 @@
 .color_value {
     display: block;
+    border: 1px solid #ccc;
+    padding: 20px;
+    margin: 5px;
+    text-align: center;
+}
+
+.xrayhtml pre {
```

Comment:
I think it will be better to put all styleguide specific styles aside, and especially this xrayhtml is from plugin, so probably it is not going to be intuitive to tell what it is to people